crates/core/src/sql/execute.rs

```diff
@@ -357,9 +358,27 @@ pub(crate) mod tests {
         let (db, _tmp_dir) = make_test_db()?;
 
         let mut tx = db.begin_tx();
-        create_table_with_rows(&db, &mut tx, "Inventory", data.inv.head.into(), &data.inv.data)?;
-        create_table_with_rows(&db, &mut tx, "Player", data.player.head.into(), &data.player.data)?;
-        create_table_with_rows(&db, &mut tx, "Location", data.location.head.into(), &data.location.data)?;
+        create_table_with_rows(
+            &db,
+            &mut tx,
+            "Inventory",
+            data.inv.head.into(),
+            &data.inv.data.iter().map(|row| row.data.clone()).collect_vec(),
```

Comment:
I think this could be refactored to take the `RelValue` and compute things inside `create_table_with_rows`